python/RbAIAnalysis.py

```diff
@@ -54,10 +55,10 @@ def analyzeMeasurement(self, measResults, iterationResults, experimentResults):
             MOTX = X1/X2 # x1-x2
             MOTY = Y1/Y2 # y1-y2
             MOTZ = Z1/Z2 # z1-z2
-            print('X1/X2 = ', MOTX," Total X Power: ", X1+X2 )
-            print('Y1/Y2 = ', MOTY," Total Y Power: ", Y1+Y2 )
-            print('Z1/Z2 = ', MOTZ," Total Z Power: ", Z1+Z2 )
-            print('TOTAL = %s uW' % MOTtot)
+            logger.info('X1/X2 = ', MOTX," Total X Power: ", X1+X2 )
```

Comment:
This does not work, we get the following error: 
Traceback (most recent call last):
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 853, in emit
    msg = self.format(record)
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 726, in format
    return fmt.format(record)
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\User\lib\site-packages\colorlog\colorlog.py", line 129, in format
    message = super(ColoredFormatter, self).format(record)
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 465, in format
    record.message = record.getMessage()
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 329, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Logged from file RbAIAnalysis.py, line 58
Traceback (most recent call last):
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 853, in emit
    msg = self.format(record)
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 726, in format
    return fmt.format(record)
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 465, in format
    record.message = record.getMessage()
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 329, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Logged from file RbAIAnalysis.py, line 58
Traceback (most recent call last):
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 853, in emit
    msg = self.format(record)
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 726, in format
    return fmt.format(record)
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 465, in format
    record.message = record.getMessage()
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 329, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Logged from file RbAIAnalysis.py, line 58
Traceback (most recent call last):
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 853, in emit
    msg = self.format(record)
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 726, in format
    return fmt.format(record)
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\User\lib\site-packages\colorlog\colorlog.py", line 129, in format
    message = super(ColoredFormatter, self).format(record)
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 465, in format
    record.message = record.getMessage()
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 329, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Logged from file RbAIAnalysis.py, line 59
Traceback (most recent call last):
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 853, in emit
    msg = self.format(record)
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 726, in format
    return fmt.format(record)
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 465, in format
    record.message = record.getMessage()
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 329, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Logged from file RbAIAnalysis.py, line 59
Traceback (most recent call last):
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 853, in emit
    msg = self.format(record)
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 726, in format
    return fmt.format(record)
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 465, in format
    record.message = record.getMessage()
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 329, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Logged from file RbAIAnalysis.py, line 59
Traceback (most recent call last):
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 853, in emit
    msg = self.format(record)
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 726, in format
    return fmt.format(record)
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\User\lib\site-packages\colorlog\colorlog.py", line 129, in format
    message = super(ColoredFormatter, self).format(record)
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 465, in format
    record.message = record.getMessage()
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 329, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Logged from file RbAIAnalysis.py, line 60
Traceback (most recent call last):
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 853, in emit
    msg = self.format(record)
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 726, in format
    return fmt.format(record)
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 465, in format
    record.message = record.getMessage()
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 329, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Logged from file RbAIAnalysis.py, line 60
Traceback (most recent call last):
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 853, in emit
    msg = self.format(record)
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 726, in format
    return fmt.format(record)
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 465, in format
    record.message = record.getMessage()
  File "C:\Users\Rb\AppData\Local\Enthought\Canopy\App\appdata\canopy-1.7.3.3335.win-x86_64\lib\logging\__init__.py", line 329, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Logged from file RbAIAnalysis.py, line 60


We attempted to just at str() in to the values, but that did not work